src/SFA.DAS.EmployerAccounts.UnitTests/Mappings/WhenAutoMapping.cs

```diff
@@ -17,7 +17,6 @@ public void ThenShouldUseValidConfiguration()
                 c.AddProfile<AgreementMappings>();
                 c.AddProfile<EmploymentAgreementStatusMappings>();
                 c.AddProfile<LegalEntityMappings>();
-                c.AddProfile<MembershipMappings>();
```

Comment:
why has this been removed? Membership mappings are still valid?

Reply:
I'd incorrectly removed the mappings at some point and restored them but seemingly missed the test.